.editorconfig

```diff
@@ -0,0 +1,11 @@
+[*]
+charset=utf-8
+end_of_line=lf
+insert_final_newline=false
```

Comment:
I let IntellIj generate the file and drop all the extensions and settings I thought irrelevant. 

Missed this one.